tests/components/slices/NavigationZone.test.js

```diff
@@ -111,5 +111,19 @@ describe('NavigationZone slice', () => {
         expect(component.vm.isOpenDropdown(dropdownIndex)).toBeFalsy()
       })
     })
+
+    describe('#subIsActive', () => {
+      it('should return false if link.url is not the same as the current route', () => {
+        $route.path = '/tests'
+        const links = [{ link: { url: '/current' } }]
+        expect(component.vm.subIsActive(links)).toBe(false)
+      })
+
+      it('should return true if link.url is the same as the current route', () => {
```

Comment:
Une petite reformulation dans le genre ferait pas de mal non ? 
```suggestion
      it('should return true if links contains link.url same as the current route', () => {
```